src/treemodel/range.js

```diff
@@ -195,6 +250,28 @@ CKEDITOR.define( [ 'treemodel/position', 'treemodel/positioniterator', 'utils' ]
 		}
 
 		/**
+		 * Returns an iterator that iterates over all {@link treeModel.Node nodes} that are top-level nodes in this range
+		 * and returns them. A node is a top-level node when it is in the range but it's parent is not. In other words,
+		 * this function splits the range into separate sub-trees and iterates over their roots.
+		 * **Note:** You might like to use `Array.from` to convert return value of this function into array.
```

Comment:
`**Note:** You might like to use `Array.from` to convert return value of this function into array.`

This comment is true for every iterator. You could have some "How to work with ES6 API", but this does not look like a good place for it.